### PR TITLE
Update code of conduct

### PR DESCRIPTION
**Pull request by lilyminium**
I have essentially replaced the MolSSI's code of conduct with ours. Minor changes have been made to make it more general, including:

* Taking out the parts that invite people to contact us via mailing list or the anonymous form
* Taking out the parts referring to specific roles in the community (e.g. founders, developers, mentors) and spaces (e.g. Discord, mailing list, etc)

Instead, I kept the sections MolSSI has on scope and enforcement:

```
## Scope

This Code of Conduct applies both within project spaces and in public spaces
when an individual is representing the project or its community. Examples of
representing a project or community include using an official project e-mail
address, posting via an official social media account, or acting as an
appointed representative at an online or offline event. Representation of a
project may be further defined and clarified by project maintainers.

## Enforcement
Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported by contacting the project team at '{{cookiecutter.author_email}}'. The project team will
review and investigate all complaints, and will respond in a way that it deems
appropriate to the circumstances. The project team is obligated to maintain
confidentiality with regard to the reporter of an incident. Further details of
specific enforcement policies may be posted separately.
Project maintainers who do not follow or enforce the Code of Conduct in good
faith may face temporary or permanent repercussions as determined by other
members of the project's leadership.
```

Are we alright with this?

Preview at https://github.com/lilyminium/test_mda_condaforge_rtd/blob/main/CODE_OF_CONDUCT.md

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]

File: {{cookiecutter.repo_name}}/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,49 +1,99 @@
...
+     then stop
+
+6. **Moderate your expectations**. Many in our community volunteer their time.
+   They are probably not purposefully ignoring issues, refusing to engage in
+   discussion, avoiding features, etc. but often just unavailable.
+
+7. **When we disagree, try to understand why**. Disagreements, both social and
+   technical, happen all the time and MDAnalysis is no exception. It is important
+   that we resolve disagreements and differing views constructively. Remember
+   that we’re different. The strength of MDAnalysis comes from its varied community
```
Comment (orbeckst) [inline {{cookiecutter.repo_name}}/CODE_OF_CONDUCT.md:86]: Should we remove "MDAnalysis" from here? Maybe replace with the project placeholder?

**Review by orbeckst — CHANGES_REQUESTED** [review]
looks good, although I would remove any direct references to MDAnalysis, unless we want to say that as an MDAKit you essentially fall under _our_ CoC. But that would be a pretty strong "copy-left"-like CoC... and my opinion is to rather let the kit authors decide for themselves. The template is a great start for anyone launching a kit.

File: {{cookiecutter.repo_name}}/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (lilyminium) [reply {{cookiecutter.repo_name}}/CODE_OF_CONDUCT.md:86]: Oops, yes, I missed this sorry!

Comment (lilyminium) [reply {{cookiecutter.repo_name}}/CODE_OF_CONDUCT.md:86]: Now looks like:

https://github.com/lilyminium/test_mda_pip_rtd/blob/main/CODE_OF_CONDUCT.md

> Disagreements, both social and technical, happen all the time and Test Project with PyPI and ReadTheDocs is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of Test Project with PyPI and ReadTheDocs comes from its varied community that includes people from a wide range of backgrounds.

**Review by orbeckst — APPROVED** [review]

**Pull request merged**